apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -872,49 +884,38 @@ func (m *ModelManagerAPI) makeModelSummary(mi state.ModelSummary) *params.ModelS
 		summary.Counts = append(summary.Counts, params.ModelEntityCount{Entity: params.Units, Count: mi.UnitCount})
 	}
 
-	access, err := common.StateToParamsUserAccessPermission(mi.Access)
+	access, err := common.StateToParamsUserAccessPermission(mi.UserAccess)
```

Comment:
Can we change the name of this func now to something more meaningful?

Reply:
Good point, changed to `EncodeAccess`